### ISSUE TITLE
Panic on ubuntu-server / wsl

### ISSUE THREAD

**Issue by dten** [hwchen/keyring-rs#15]
If i try use this on ubuntu-server or windows subsystem for linux (wsl) / bash on windows I get the following error

```
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: SecretServiceError(Dbus(D-Bus error: Unable to autolaunch a dbus-daemon without a $DISPLAY for X11 (org.freedesktop.DBus.Error.NotSupported)))', /checkout/src/libcore/result.rs:916:4
```

is X11 really needed to access the keyring?

**Comment by hwchen** [comment]
I can investigate more; my initial usecase was not for headless systems so
I didn't think about it.

You can also check
https://github.com/jaraco/keyring/blob/master/README.rst#using-keyring-on-headless-linux-systems
and https://github.com/jaraco/keyring/issues/280. My implementation is a
port of that library, so I think that issue might help

**Comment by adobeDan** [comment]
This was fixed in release 0.10.4.  See the README for info about using the keyring on headless systems.
